src/main/java/no/sikt/nva/pubchannels/handler/create/serialpublication/CreateSerialPublicationHandler.java

```diff
@@ -47,7 +47,7 @@ protected void validateRequest(CreateSerialPublicationRequest request,
     protected SerialPublicationDto processInput(CreateSerialPublicationRequest request,
                                                 RequestInfo requestInfo, Context context) throws ApiGatewayException {
         var channelRegistryCreateRequest = ChannelRegistryCreateSerialPublicationRequest.fromClientRequest(request);
-        var response = JOURNAL.equals(request.type())
+        var response = JOURNAL.equalsIgnoreCase(request.type())
```

Comment:
Should this default to `Series`, or does it make sense to raise an exception for unexpected values?

Reply:
Nevermind, this is probably covered by the validation method.

Reply:
It validates type before we reach this point in `private void validateType(CreateSerialPublicationRequest input)`